### DISCUSSION TITLE
Is --cache options in pack go-client not support right now?

### DISCUSSION THREAD

**Discussion by inspirit941** [buildpacks/community#213] [Q&A] [answered]
from v0.28, pack cli support --cache to define cache technique for build process.
- https://github.com/buildpacks/pack/pull/1477/files
but it seems that cacheOpts struct used for --cache options is in internal package, which is not possible to import from external project.
<img width="595" alt="스크린샷 2023-04-05 오후 1 15 04" src="https://user-images.githubusercontent.com/26548454/229980025-0ec64333-d14e-462f-849e-e70247468666.png">

is there any workaround to use cacheOpts in pack go-client? or any plans to support this near future?

**Comment by natalieparellano** [comment, marked as answer]
@inspirit941 thanks for raising this - it may be worth filing an issue on pack. We've exported stuff before to help out library users: https://github.com/buildpacks/pack/issues/814, https://github.com/buildpacks/pack/issues/657
